desktop/core/src/desktop/lib/botserver/slack_client.py

```diff
@@ -15,12 +15,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
+
 from desktop import conf
 
 SLACK_VERIFICATION_TOKEN = conf.SLACK.SLACK_VERIFICATION_TOKEN.get()
 SLACK_BOT_USER_TOKEN = conf.SLACK.SLACK_BOT_USER_TOKEN.get()
 
 slack_client = None
-if conf.SLACK.IS_ENABLED.get():
-  from slack_sdk import WebClient
-  slack_client = WebClient(token=SLACK_BOT_USER_TOKEN)
\ No newline at end of file
+if conf.SLACK.IS_ENABLED.get() & sys.version_info[0] > 2:
+    from slack_sdk import WebClient
+    slack_client = WebClient(token=SLACK_BOT_USER_TOKEN)
```

Comment:
Even if the style is correct now; shall we leave as it was to reduce problems on rebases?

Reply:
Done